src/js/components/lightbox-panel.js

```
@@ -28,284 +40,305 @@ export default {
-                                    ({width, height}) => setIframe(width, height),
-                                    setIframe
-                                );
-                            } else {
-                                setIframe(width, height);
-                            }
-                        },
-                        setIframe
+                } else if (
+                    (matches = src.match(
+                        /\/\/(?:.*?youtube(-nocookie)?\..*?(?:[?&]v=|\/shorts\/)|youtu\.be\/)([\w-]{11})[&?]?(.*)?/,
+                    ))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)